Modules/System/SharePoint/src/model/listitem/SharePointListItem.Table.al

```diff
@@ -0,0 +1,87 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about list item entity.
+/// </summary>
+table 9103 "SharePoint List Item"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint List Item';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; Guid; Guid)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Id';
+        }
+
+        field(2; Title; Text[250])
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Title';
+        }
+
+        field(3; Created; DateTime)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Created';
+        }
+
+        field(4; Attachments; Boolean)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Attachments';
+        }
+
+        field(5; "File System Object Type"; Integer)
+        {
+            //Enum with negative values
+            // Invalid -	Enumeration whose values specify whether the object is invalid. The value = -1.
+            // File	- Enumeration whose values specify whether the object is a file. The value = 0.
+            // Folder	- Enumeration whose values specify whether the object is a folder. The value = 1.
+            // Web - Enumeration whose values specify whether the object is a site. The values = 2.
+            DataClassification = SystemMetadata;
+            Caption = 'File System Object Type';
+        }
+
+        field(6; "Content Type Id"; Text[250])
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Content Type Id';
+        }
+
+        field(7; Id; Integer)
```

Comment:
Not sure what this "Id" represents, but maybe call it "Item Id"?
The above "Guid" field is just a weird name to have as a primary key.

Reply:
Hi, as agreed we will keep the SharePoint naming.